packages/components/src/molecules/Carousel/CarouselItem.tsx

```diff
@@ -39,7 +40,7 @@ function CarouselItem({
       style={style}
       data-fs-carousel-item
       aria-roledescription="slide"
-      id={`carousel-item-${index}`}
+      id={`${id}-carousel-item-${index}`}
```

Comment:
We also need to add it to bullets 😬

Reply:
done here https://github.com/vtex/faststore/pull/1737#:~:text=useId%20as%20fallback-,f8fd76c,-Merge%20branch%20%27fix